### DISCUSSION TITLE
indicators parameters superset

### DISCUSSION THREAD

**Discussion by carlok** [vsaveris/trading-technical-indicators#18] [Q&A] [answered]
Hi, I still have to try your library but I think it's a huge and useful work. Thanks.
I have a question so I don't open an issue.
The indicators are a lot so I didn't check each of them.
Can the user assume that [high, low, close, volume] is the largest set an indicator could require as input or are there indicators requiring extra inputs?
Thanks.

**Comment by vsaveris** [comment, marked as answer]
Hi,

yes you can assume that the largest set an indicator could require is the `(high, low, open, close, volume)`. 

In the documentation is stated what is required per each indicator, but you can have a quicker look in the [indicators properties](https://github.com/vsaveris/trading-technical-indicators/blob/master/tti/indicators/properties/indicators_properties.py) (look for the value of the `required_input_data` for the indicator you are interested.

Please note that in an hour the first stable release 0.2.0 will be available in the PyPi, and you can download this through pip. It includes some good updates comparing to the beta releases, so it is worth to start with it.

Vasilis

**Reply by carlok** [reply]
Perfect. The huge dictionary of indicators answers to my second question too (where to programmatically find all the indicators). So I could also assume that passing always the whole `(high, low, open, close, volume)` set, even when some of them are not used, would not harm the called indicator, right?
Yes, I'll wait 0.2.0.
Thanks again.

**Reply by vsaveris** [reply]
Yes, it won't harm. The code checks just for the required columns.

**Comment by vsaveris** [comment]
version 0.2.0 is released and you can install it from now on.

You can find all the indicators implemented in the [tti.indicators package init file](https://github.com/vsaveris/trading-technical-indicators/blob/master/tti/indicators/__init__.py)

Documentation can be found in [tti package documentation](https://trading-technical-indicators.readthedocs.io/en/latest/index.html)

Project site is [project site](https://www.trading-technical-indicators.org/)

For a quick start you can see the example in the documentation [example](https://trading-technical-indicators.readthedocs.io/en/latest/tti_indicators_examples.html)

Feel free to share your feedback.

Vasilis
